lib/node_modules/@stdlib/utils/circular-buffer/docs/types/test.ts

```diff
@@ -1,35 +1,64 @@
 /*
-* @license Apache-2.0
-*
-* Copyright (c) 2021 The Stdlib Authors.
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*/
+ * @license Apache-2.0
```

Comment:
These indentation changes should be undone. We strive for consistency throughout the code base.

Note that this may mean toggling a setting in your code editor if your editor auto-indents Doxygen style comments.